cmd/detectExecuteScan.go

```diff
@@ -1044,6 +1044,7 @@ func logConfigInVerboseMode(config detectExecuteScanOptions) {
 	config.Token = "********"
 	config.GithubToken = "********"
 	config.PrivateModulesGitToken = "********"
+	config.RepositoryPassword = "********"
```

Comment:
Is this somehow related?

Reply:
Nope, it showed up because this branch and https://github.com/SAP/jenkins-library/pull/4901 were based on a different commit to master. This change is gone now.